src/Controller/HomeController.php

```diff
@@ -1,24 +0,0 @@
-<?php
-
-namespace App\Controller;
```

Comment:
Why are you deleting HomeController ?

Reply:
> Why are you deleting HomeController ?

Just because I do not need it anymore, and to avoid any route troubles which might appear after Viktor finished his index page task.

Reply:
if you merge it will delete it from main branch